src/main/resources/static/css/button.css

```diff
@@ -35,4 +35,8 @@
 
 #errorModal .close {
     margin-top: -0.3rem;
-}
\ No newline at end of file
+}
+#gradeInput{
```

Comment:
why do we need this?

Reply:
guess that time it was needed but now it seems to work fine without it